lib/run.js

```diff
@@ -150,9 +150,9 @@ function run (opts, cb) {
         finish: new Date(),
         connections: opts.connections,
         pipelining: opts.pipelining,
-        '2xx': statusCodes[1],
         'non2xx': statusCodes[0] + statusCodes[2] + statusCodes[3] + statusCodes[4]
```

Comment:
It's used in tests, and its not very harmful to keep it in, I can remove if you prefer though